### PR TITLE
f-button@v1.1.1 - Minor fixes.

### PR DESCRIPTION
**Pull request by DamianMullins**
### Changed
- Formatting in storybook files.

### Fixed
- Pass button attributes to component correctly.
- CSS class name spelling mistake.

### CODE REVIEW

File: packages/components/atoms/f-button/src/components/Button.vue [hunk 1]
```diff
@@ -8,9 +8,9 @@
             $style[`o-btn--size${buttonSizeClassname}`],
             (isFullWidth ? $style['o-btn--fullWidth'] : '')
         ]"
-        :attributes="$attrs"
         :action-type="buttonActionType"
         :data-test-id="`${componentType}-component`"
+        v-bind="$attrs"
```
Comment (minrwhite) [inline packages/components/atoms/f-button/src/components/Button.vue]: As the resolved `component` element ends up being the root element of the `Button` component, and we haven't set [`inheritAttrs: false`](https://vuejs.org/v2/guide/components-props.html#Disabling-Attribute-Inheritance) for `Button`, is it necessary to set this?

**Review by ashleynolan — DISMISSED** [review]

File: packages/components/atoms/f-button/src/components/Button.vue (hunk 1 above)
Comment (DamianMullins) [reply packages/components/atoms/f-button/src/components/Button.vue]: Looks like it isn't, I have removed this now, thanks!

**Review by minrwhite — APPROVED** [review]

**Review by AlixRichards — APPROVED** [review]

**Review by zlatinivanov-je — APPROVED** [review]
Should we have some rules in place for code formatting, some of the changes here look like could change based on personal preferences. Or are there any in the project already?

**Comment by DamianMullins** [comment]
I tend to follow the linting rules and maybe some of them are no longer enforced but I still follow them such as object and import properties on new lines, as well as whitespace between object properties - I find the code easier to scan when there is some breathing room between lines.

I can have a dig around to see if rules exist for that type of thing.

**Pull request merged**